src/microbe_stage/BioProcess.cs

```diff
@@ -29,6 +29,8 @@ public class BioProcess : IRegistryType
 
     public string InternalName { get; set; } = null!;
 
+    public bool Active { get; set; } = true;
```

Comment:
This cannot be added here, BioProcess is similar to OrganelleDefinition as it is a registry data type. It may not be modified once the game has started up. The speed control needs to be put into TweakedProcess. And if I remember right the tweaked process list is not saved, which will need to be changed with this change for it to persist. Also the method to rebuild the tweaked process list should copy any adjusted speeds when it makes the new list, otherwise the speed control state would get reset each time the player grows a new organelle.